docker/test-php/Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM sillsdev/web-languageforge:base-php-7.4
+FROM sillsdev/web-languageforge:base-php
 
 
 # ----- LINES BELOW COPIED FROM APP DOCKERFILE ----------
 WORKDIR /var/www/html
 ENV COMPOSER_ALLOW_SUPERUSER=1
-RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/* \
-    && install-php-extensions @composer
 COPY src/composer.json src/composer.lock /var/www/html/
 RUN composer install
 
-RUN install-php-extensions xdebug
+RUN install-php-extensions xdebug-^3.1
```

Comment:
For whatever reason this didn't specify the xdebug version as we do in the development app build